adagrams/game.py

```diff
@@ -1,11 +1,82 @@
+import random
+import string
+
+value_of_letters = {
+        
+        "A" : 1 , "B" : 3 , "C": 3 , 
+        "D" : 2 , "E": 1 , "F": 4 ,
+        "G" : 2 , "H" : 4, "I" : 1, 
+        "J" : 8 , "K" : 5 ,"L" : 1 , 
+        "M" : 3 , "N" : 1 , "O" : 1 , 
+        "P": 3, "Q" : 10 , "R" : 1 ,
+        "S" : 1 , "T" : 1 , "Q" : 10 ,
+        "U" : 1 , "V" : 4 , "W" : 4 , 
+        "X" : 8 , "Y" : 4 , "Z" : 10
+        
+        }
+    
+
 def draw_letters():
-    pass
+    
+    pool_of_letters = {
+    
+    
+    "A" : 9, "N" : 6, "B" : 2, "O" : 8, "C" : 2, "P" : 2, "D" : 4, "Q" : 1, 
+    "E" : 12,"R" : 6, "F" : 2, "S" : 4, "G" : 3, "T" : 6, "H" : 2, "U" : 4,
+    "I" : 9, "V" : 2, "J" : 1, "W" : 2, "K" : 1, "X" : 1, "L" : 4, "Y" : 2, 
+    "M" : 2, "Z" : 1 
+    
+    }
+    
+    
+    
+    player_letters = []
+    random_letter = ""
+    
+    while len(player_letters) < 10:
+        random_letter = random.choice(string.ascii_uppercase)
```

Comment:
A couple of thoughts about line 37: 

1) Nice job using the built-in function random.choice(). How does this function work under the hood?

I encourage students to practice writing this kind of logic on their own because it provides practice in thinking through the algorithm to grab a random letter and provides the muscle memory of writing loops and using random too.

While writing it out on your own is more lines of code, I think that's ok for initial projects to you get this practice. Especially in interviews, we should feel confident that we understand how built-in functions work under the hood because your interviewer may ask you to explain it to them.

2) This doesn't quite match the requirement in the README. `string.ascii_uppercase` returns a string that has 26 letters A-Z in it. How can you use `pool_of_letters` to create a list or a string with 9 As, 2 Bs, 2 Cs, etc..?